network/remove_peer.js

```diff
@@ -137,34 +139,66 @@ module.exports = (args, cbk) => {
         cbk);
       }],
 
-      // Check channels for peer to make sure that they can be cleanly closed
-      checkChannels: ['getChannels', ({getChannels}, cbk) => {
+      //Select channel to close if multiple are available
+      selectedChannel: ['getChannels', ({getChannels}, cbk) => {
         // Exit early when a peer is not specified or force closing is OK
         if (!args.public_key || !!args.is_forced) {
           return cbk();
-        }
+        } 
 
-        const selectedChannels = getChannels.channels.filter(channel => {
+        const channels = getChannels.channels.filter(channel => {
           // Ignore channels that are not the specified public key
           if (channel.partner_public_key !== args.public_key) {
             return false;
           }
 
-          // Exit early when there are no outpoints, consider all peer channels
-          if (!args.outpoints.length) {
-            return true;
-          }
+          //Return channels with the peer
+          return true;
+        });
 
-          // Only include selected channels
-          return args.outpoints.includes(asOutpoint(channel));
+        // Exit early when no channels are available
+        if (!channels.length) {
+          return cbk([404, 'NoChannelsToCloseWithPeer']);
+        }
+
+        //Exit early if not selecting a channel or only one is available
+        if (channels.length === 1 || !args.selectChannels) {
+          return cbk(null, channels);
+        }
+
+        const choices = channels.map(channel => {
+          return {
+            name: `Channel Id: ${channel.id}, Inbound/Outbound: ${tokensAsBigUnit(channel.remote_balance)} / ${tokensAsBigUnit(channel.local_balance)},  Est Disk Usage: ${estimateDiskFootprint(channel.past_states)}`,
```

Comment:
this doesn't work with force closing super well - let's say one of my channels is inactive, then I go to select the inactive channel to close, it will error later on, now I think oh I need to force flag on that, I go back up and put --select-channels --force and then it doesn't show me the selection UI and it force closes all the channels

Reply:
A potential solution is show the channel picker even when `--force` is specified, but when `--force` isn't specified show the ineligible channels and mark them disabled in the choices, maybe with an emoji hint to why it's ineligible

This flow would mean the user would see the channels disabled and they have a sign that they are disabled or in-use, then they can run the command again with --force --select-channels to override the disable

Reply:
makes sense, will fix it.

Reply:
Please check now.